### ISSUE TITLE
Add the expo file upload to send the image

### ISSUE THREAD

**Issue by VukIG** [VukIG/Melanoma-Detector#13]

**Comment by VukIG** [comment]
Using this tutorial https://www.youtube.com/watch?v=pBEYprNAs4c&t=325s figure out a way to send the picture as a file to the server. This is a front-end issue.Take a look at the main.py of the server folder and make sure that the predict function takes in adequate arguments. Make sure to test out the server and see if requesting the front end works

**Comment by riiich** [comment]
So, instead of base64 we want the actual image file?

**Comment by VukIG** [comment]
Yep, I pushed the changes to the main.py so It works rn, you just need to do the front end part

**Comment by fawzii0x3** [comment]
I think support to get File type would be good 
I'm using Apollo client cuz I couldn't figure out an alternative to ReactNativeFile 
So I can use urql 🙃
